### DISCUSSION TITLE
Favorite IDE

### DISCUSSION THREAD

**Discussion by revwines** [wheels-dev/wheels#1207] [Polls]
After a long break from CFML, I'm taking on a small project.

So what IDE is popular these days?

**Comment by dbelanger** [comment]
These days, [VS Code](https://code.visualstudio.com/) (https://code.visualstudio.com/) with the following extensions:

- CFML (KamasamaK)
- Auto Close Tag (Jun Han)

**Reply by mikegrogan** [reply]
Yeah, just get VS Code. It's arguably the best and easiest to get going with.

**Reply by blastiko** [reply]
There is also an extension named "Adobe ColdFusion Builder", with RDS functionality, but unfortunately it is "gaunt", few integrated with VS settings and with some bug.

**Comment by adamcameron** [comment]
No mention of VSCode? This reads like a poll created a decade ago...

**Reply by revwines** [reply]
haha that's about the last time I coded CFML

**Comment by bpamiri** [comment]
I don’t do much Adobe ColdFusion work any more. I like the RDS features of the old bloated CFML IDEs when I was using Adobe CF. But now that I am mostly using CommandBox and Luce I use VS Code exclusively. 

Sent from my iPad
-Peter Amiri
 m 949 338 3862
